src/main/java/core/basesyntax/strategy/operationstrategy/OperationCalculatorHandler.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.strategy.operationstrategy;
+
+import core.basesyntax.model.FruitTransaction.Operation;
+import core.basesyntax.strategy.operationstrategy.impl.BalanceOperationCalculatorImpl;
+import core.basesyntax.strategy.operationstrategy.impl.PurchaseOperationCalculatorImpl;
+import core.basesyntax.strategy.operationstrategy.impl.ReturnOperationCalculatorImpl;
+import core.basesyntax.strategy.operationstrategy.impl.SupplyOperationCalculatorImpl;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class OperationCalculatorHandler {
+    private static final Map<Operation, OperationCalculator> operationCalculators = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Operation, OperationCalculator> operationCalculatorsMap;
```
initialize using constructor